Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/GridSplitter.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// <summary>
     /// Represents the control that redistributes space between columns or rows of a Grid control.
     /// </summary>
-    public partial class GridSplitter : Control
+    public partial class GridSplitter : SplitBase
     {
         internal const int GripperCustomCursorDefaultResource = -1;
```

Comment:
None of these cursor properties should be needed anymore?

Reply:
I occasionally encountered the known issue with the cursor semi-permanently changing to horizontal arrows (see the video in #3949), but wasn't able to consistently reproduce.  I'll still go ahead and trim out the cursor code a bit more.